pkg/velerorunner/task.go

```diff
@@ -109,3 +123,91 @@ func (t *Task) getSourceClient() (k8sclient.Client, error) {
 func (t *Task) getDestinationClient() (k8sclient.Client, error) {
 	return t.PlanResources.DestMigCluster.GetClient(t.Client)
 }
+
+// Log the task based on phase.
+func (t *Task) logPhase() {
+	log := t.Log
+	switch t.Phase {
+	case "":
+		log.Info(
+			"Task has started.",
+			"owner",
+			t.Owner.GetName())
+	case RestartRestic:
+		log.Info(
+			"Restart Restic pods.",
+			"owner",
+			t.Owner.GetName())
+	case WaitOnResticRestart:
+		log.Info(
+			"Waiting on Restic pod restart to complete.",
+			"owner",
+			t.Owner.GetName())
+	case ResticRestartCompleted:
+		log.Info(
+			"Restic pod has restart completed.",
+			"owner",
+			t.Owner.GetName())
+	case BackupStarted:
+		log.Info(
+			"Backup started.",
+			"owner",
+			t.Owner.GetName(),
+			"backup",
+			t.Backup.Name)
+	case BackupCompleted:
+		log.Info(
+			"Backup has completed.",
+			"owner",
+			t.Owner.GetName(),
+			"backup",
+			t.Backup.Name)
+	case BackupFailed:
+		log.Info(
+			"Backup has failed.",
+			"owner",
+			t.Owner.GetName(),
+			"backup",
+			t.Backup.Name)
+	case WaitOnBackupReplicated:
+		log.Info(
+			"Waiting Backup replication to destination.",
```

Comment:
NIT: "Waiting for Backup to replicate to destination", I don't like mine much better but grammar here is off